pages/dkp/konvoy/2.2/choose-infrastructure/vsphere/air-gapped/create-bastion-vm/index.md

```diff
@@ -14,21 +14,21 @@ Before you start this procedure, ensure that you complete the [Prerequisites][pr
 ## Create a vSphere Bastion VM Host Template
 When creating an air-gapped vSphere cluster, the bastion VM hosts the installation DKP Konvoy bundles and images, and the Docker registry, needed to create and operate your vSphere cluster. The bastion VM must have access to the vSphere API Server (vCenter Server).
 
-1.  Create a bastion VM host template for the cluster nodes to use within the air-gapped network. This bastion VM host also needs access to a Docker registry in lieu of an Internet connection for pulling Docker images. The recommended template naming pattern is `../folder-name/dkp-e2e-bastion-template' or similar.
+1.  Create a bastion VM host template for the cluster nodes to use within the air-gapped network. This bastion VM host also needs access to a Docker registry in lieu of an Internet connection for pulling Docker images. The recommended template naming pattern is `../folder-name/dkp-e2e-bastion-template` or similar.
 
 1.  Find and record the bastion VM's IP or host name.
 
 1.  Download the required Konvoy binaries and installation bundles directly to the bastion host, or transfer them using your environment's approved methods:
 
-    - [downloads.mesosphere.io/konvoy/konvoy_v2.2.0-rc.1_checksums.txt][http://downloads.mesosphere.io/konvoy/konvoy_v2.2.0-beta.1_checksums.txt]
+    - [downloads.mesosphere.io/konvoy/konvoy_v2.2.0-rc.1_checksums.txt](http://downloads.mesosphere.io/konvoy/konvoy_v2.2.0-beta.1_checksums.txt)
```

Comment:
Couple things here.   First is that we are not supposed to reference 'downloads.mesosphere.io' anywhere.  Everything should be "downloads.d2iq.com".   This is an edict from on high.

Second, I think rather than put all these links here, we should just refer the user to the download tab (which sends them to the support site), and instead just mention the files they need to collect here:

```suggestion
    - konvoy_v2.2.0-rc.1_checksums.txt
```

Reply:
Good to know in the mseo.io thing, and better approach.